### PR TITLE
[Bugfix] Travis config for deployment

### LINKED ISSUE #708: Problem with @ng-toolkit/_utils
I really don't know what's the issue. Probably `@ng-toolkit/_utils` it's not published in the proper way. Unfortunately, all the packages which depend on those, are unable to properly consume this package, which leads to:

```
Error: Cannot find module '@ng-toolkit/_utils/testing
```

The problem doesn't occur on my local environment (by running ./scripts/deploy_verdaccio.sh), it occurs only on the Travis CI.

### PR DESCRIPTION
**Pull request by KingDarBoja**
It is supposed to fix #708. I do get the same travis error on my local machine, because each package is looking for `@ng-toolkits/_utils` but at the wrong version (*8.0.0*) instead of the published one (*8.0.1*).

Still working on the fix.

### CODE REVIEW

**Comment by KingDarBoja** [comment]
@maciejtreder I think the issue is being triggered by this missing script `"prepare": "npm test",` at the `package.json` file on `@ng-toolkit/utils`. That package is not being built before publishing but still should be able to look at the index file at testing folder.

**Comment by KingDarBoja** [comment]
@maciejtreder Can confirm, it was the missing build step for the `_utils` package. Got success output for publishing `serverless` package on my local enviroment.

```
--- NPM INSTALL COMPLETED FOR serverless ---
index.js  index.js.map  index.ts  package.json  publish.sh  publish_verdaccio.sh  testing  tsconfig.json

> @ng-toolkit/serverless@8.0.3 prepare .
> npm test


> @ng-toolkit/serverless@8.0.3 test C:\Users\mbojatop\Desktop\atom_project\ng-toolkit\schematics\serverless
> npm run build && jasmine src/**/*_spec.js


> @ng-toolkit/serverless@8.0.3 build C:\Users\mbojatop\Desktop\atom_project\ng-toolkit\schematics\serverless
> tsc -p tsconfig.json

Randomized with seed 42444
Started
[bugsnag] Loaded!
.........


9 specs, 0 failures
Finished in 1.679 seconds
Randomized with seed 42444 (jasmine --random=true --seed=42444)
npm notice
npm notice package: @ng-toolkit/serverless@8.0.3
npm notice === Tarball Contents ===
npm notice 1.3kB  package.json
npm notice 273B   collection.json
npm notice 546B   postinstall.js
npm notice 84.8kB project_id.png
npm notice 44B    publish_verdaccio.sh
npm notice 18B    publish.sh
npm notice 14.0kB README.md
npm notice 73B    tsconfig.json
npm notice 888B   src/files/aws/lambda.js
npm notice 735B   src/files/aws/serverless-aws.yml
npm notice 234B   src/files/common/local.js
npm notice 562B   src/files/common/server_static.ts
npm notice 1.3kB  src/files/common/server_universal.ts
npm notice 1.1kB  src/files/common/webpack.server.config.js
npm notice 358B   src/files/firebase/functions/index.js
npm notice 372B   src/files/gcloud/index.js
npm notice 496B   src/files/gcloud/serverless-gcloud.yml
npm notice 12.5kB src/index_spec.js
npm notice 5.8kB  src/index_spec.js.map
npm notice 12.2kB src/index_spec.ts
npm notice 19.6kB src/index.js
npm notice 13.0kB src/index.js.map
npm notice 19.4kB src/index.ts
npm notice 980B   src/schema.d.ts
npm notice 930B   src/schema.json
npm notice === Tarball Details ===
npm notice name:          @ng-toolkit/serverless
npm notice version:       8.0.3
npm notice package size:  89.6 kB
npm notice unpacked size: 191.5 kB
npm notice shasum:        5e83bafc440df81f4f0a4821f5736766d5d82ff1
npm notice integrity:     sha512-S3HFNrXnwz0qB[...]qG6DrUCXCw3hA==
npm notice total files:   25
npm notice
+ @ng-toolkit/serverless@8.0.3
e\[1;33m*** serverless PASSED ***
```

Ignore the version number, I was testing the `_utils` but couldn't update it if using the same version locally.

Also, I noticed the `.npmignore` files inside each package doesn't seem to work, the `tsconfig.json` files should not be included on the deployment. The other typescript files (most index.ts) acts as type definitions, right?

File: scripts/deploy_verdaccio.sh [hunk 1]
```diff
@@ -1,28 +1,33 @@
...
+for i in "${CATALOGS[@]}"; do
+    center "$i"
     cd $i
-    npm install
-    echo 'INSTALLED INSTALLED'
-    ls node_modules/@ng-toolkit/_utils/testing
-    if ./publish_verdaccio.sh
-    then
-        echo $i passed
+    npm install --verbose
```
Comment (maciejtreder) [inline scripts/deploy_verdaccio.sh]: I think that this is not necessary, as long as particular packages scripts take care about that. I think I added this line for debugging purposes.

Comment (KingDarBoja) [reply scripts/deploy_verdaccio.sh]: You mean the `npm install` part? I do find helpful to echo the current package being installed and the verbose mode was for debugging purpose, to check out if there was anything bad installed with the package dependencies. At the end, everything was okay.

Also ignore the center thing, it is to divide sections, so we can scroll or jump to any package step by looking at the colorful header.

**Pull request merged**